src/EPR.Calculator.Frontend/Controllers/UploadFileProcessingController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Net;
+using EPR.Calculator.Frontend.Models;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+
+namespace EPR.Calculator.Frontend.Controllers
+{
+    public class UploadFileProcessingController : Controller
+    {
+        private readonly IConfiguration _configuration;
+        private readonly HttpClient _client;
+
+        public UploadFileProcessingController(IConfiguration configuration, HttpClient client)
```

Comment:
Using HttpClient vs HttpClientFactory are listed here - https://learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests
If there is no time to do this now. Please create a Tech debt ticket for it. 
We should go with HttpClientFactory

Reply:
@udaydefra - I have created a tech debt ticket https://dev.azure.com/defragovuk/RWD-CPR-EPR4P-ADO/_workitems/edit/423514